### PR TITLE
Apply new review rules (1.2.0)

### PR DESCRIPTION
**Pull request by PedroHLC**
#### :thinking: What?

- Update to https://github.com/PaackEng/elm-review-config/pull/9

#### :man_shrugging: Why?

- New optimization to our codebase.

#### :pushpin: Jira Issue

Not tracked.

#### :no_good: Blocked by

- https://github.com/PaackEng/elm-review-config/pull/9

#### :clipboard: Pending

- Elm teammates' approval;
- A bump in https://github.com/PaackEng/elm-validate-action

### CODE REVIEW

File: showcase/src/Main.elm [hunk 1]
```diff
@@ -48,12 +47,6 @@ import UIExplorer.Plugins.Tabs.Icons as TabsIconsPlugin
 import Utils exposing (ExplorerModel)
 
 
-{-| importing UI.Document mostly so they compile too
--}
-type alias CompileDocument =
```
Comment (PedroHLC) [inline showcase/src/Main.elm:53]: This was once interesting because I was developing `UI.Document`...

File: showcase/src/Menu/Stories.elm [hunk 2]
```diff
@@ -24,10 +24,8 @@ import Utils
 
 
 update : Menu.Msg -> Menu.Model -> Return Msg.Msg Menu.Model
-update msg model =
-    case msg of
-        Menu.ToggleMenu ->
-            ( { model | isVisible = not model.isVisible }, Cmd.none )
+update _ model =
```
Comment (PedroHLC) [inline showcase/src/Menu/Stories.elm:27]: I've made this change by hand.

File: src/UI/Internal/Tables/Paginator.elm [hunk 3]
```diff
@@ -93,11 +92,6 @@ withAmountByPage pageAmount (Paginator prop opt) =
     Paginator prop { opt | amountByPage = pageAmount }
 
 
-withPageAmountOptions : List Int -> Paginator msg -> Paginator msg
-withPageAmountOptions pageAmountOptions (Paginator prop opt) =
```
Comment (PedroHLC) [inline src/UI/Internal/Tables/Paginator.elm:97]: @kress95 did you plan to use this?

File: src/UI/Text.elm [hunk 4]
```diff
@@ -252,9 +252,7 @@ multiline : (String -> Text) -> List String -> Text
 multiline style lines =
     let
         newSpans =
-            lines
-                |> List.map (style >> Internal.getSpans)
-                |> List.concat
+            List.concatMap (style >> Internal.getSpans) lines
```
Comment (PedroHLC) [inline src/UI/Text.elm:255]: :surfer: performance++;

**Review by niqt — APPROVED** [review]

**Review by kress95 — APPROVED** [review]

**Pull request merged**